CMakeLists.txt

```diff
@@ -128,6 +128,11 @@ else()
     message(STATUS "Build type is not set")
 endif()
 
+if (WIN32)
+    # oneDPL is incompatible with min/max macros defined in windows.h, NOMINMAX must be defined to avoid them
+    target_compile_options(oneDPL INTERFACE -DNOMINMAX)
```

Comment:
Minor suggestion: Should we add some message here for the user (e.g. "Defining NOMINMAX to avoid collisions with Windows macros.")